catalyst/dl/runner/supervised.py

```diff
@@ -83,7 +83,7 @@ def _process_output_none(self, output: Mapping[str, Any]):
         return output
 
     def predict_batch(self, batch: Mapping[str, Any]):
-        output = self._process_input(batch)
+        output = self._process_input(self._batch2device(batch, self.device))
```

Comment:
why do we need it?
this already done in https://github.com/catalyst-team/catalyst/blob/c2be013a6884fb55c5fa343dbbd7dc967d2ca0b5/catalyst/dl/core/runner.py#L102

Reply:
or you suppose to use this method for per batch inference? 🤔

Reply:
> or you  propose to use this method for per batch inference?
Exactly!

Reply:
could you please use:
```
batch = self._batch2device(batch, self.device)
output = self._process_input(batch)
```
it would be a little bit easier to read

Reply:
or, as even better solution, could you add `_predict_batch` method, that would do only model forward, use it in `Runner. _run_batch`
and add additional `predict_batch` method based on `_batch2device` and `_predict_batch`

this should remove duplication of `_batch2device` usage during training (which is more important, I guess)